src/common/SPELLS_DEMON_HUNTER.js

```diff
@@ -31,12 +31,12 @@ export default {
     icon: 'spell_warlock_soulburn',
   },
 
-  // Racial:
-  ARCANE_TORRENT: {
-    id: 202719,
-    name: 'Arcane Torrent',
-    icon: 'spell_shadow_teleport',
-  },
+	// Racial:
+	ARCANE_TORRENT_DH: {
+		id: 202719,
+		name: 'Arcane Torrent',
+		icon: 'spell_shadow_teleport',
```

Comment:
I take it this version is special because it gives DH resources?

Reply:
http://www.wowhead.com/spell=202719/arcane-torrent Fury, I think this spell should be called `ARCANE_TORRENT_FURY` instead.

Reply:
The spell details in that wowhead link imply it generates both Fury and Pain... maybe `ARCANE_TORRENT_DH` actually is a reasonable name?

Reply:
`Silence all enemies within 8 yards for 2 sec and generate 15 Fury.  Non-player victim spellcasting is also interrupted for 3 sec.`

tooltip though :(